cypress/e2e/page/categorization-pagetests.cy.js

```diff
@@ -36,7 +36,12 @@ describe("tests on categorization page ui via programmatic state loading and sto
                 // when a page is 'visited' by Cypress
                 cy.window().its("$nuxt.$router").then(router => {
 
+                    // A. Route to categorization page
```

Comment:
just out of curiosity: if you had "clicked" on the "Categorization page" nav-bar link, would that come out to the same thing here?

Reply:
Yes. This is would be akin to (though not identical to) a more traditional e2e test (i.e. `simple-e2etest.cy.js`) where instead of loading state we navigate through the UI.

I just tried it out and it works!

I am calling support function `nextPageByNav` (which is a wrapper for `cy.get(`[data-cy='menu-item-${p_navItemName}'] a`).click();`) right after the `loadTestDataIntoStore` in the `beforeEach` setup function. This also eliminates the need to make a call to the Nuxt `router` off of the `window` object.

Reply:
No strong opinion. I think we started this because of the advice on the cypress docs to not set state via the UI. I have to admit I still don't fully understand why that'd be an antipattern for us. Maybe it makes a bigger difference when you do it for external services (e.g. doing a Google-OAuth process via the UI every time you want to login?). 

All else equal, I'd go with the version that you think is more maintainable / easier to reason about when reading the test.

Reply:
I'm going to stick with the programmatic loading of state, but it's good to keep in mind that's an option.